rust/htp/README.md

```diff
@@ -0,0 +1,67 @@
+# LibHTP
+
+---
+
+Copyright 2009-2010 Open Information Security Foundation  
+Copyright 2010-2013 Qualys, Inc.
+
+---
+
+LibHTP is a security-aware parser for the HTTP protocol and the related bits
+and pieces. The goals of the project, in the order of importance, are as
+follows:
+
+ 1. Completeness of coverage; LibHTP must be able to parse virtually all
+    traffic that is found in practice.
+
+ 2. Permissive parsing; LibHTP must never fail to parse a stream that would
+    be parsed by some other web server.
+
+ 3. Awareness of evasion techniques; LibHTP must be able to detect and
+    effectively deal with various evasion techniques, producing, where
+    practical, identical or practically identical results as the web
+    server processing the same traffic stream.
+
+ 4. Performance; The performance must be adequate for the desired tasks.
+    Completeness and security are often detrimental to performance. Our
+    idea of handling the conflicting requirements is to put the library
+    user in control, allowing him to choose the most desired library
+    characteristic.
+
+ | STATUS      LIBHTP IS VERY YOUNG AT THIS POINT. IT WILL BE SOME TIME BEFORE  
+ |             IT CAN BE CONSIDER COMPLETE. AT THE MOMENT, THE FOCUS OF DEVELOPMENT  
+ |             IS ON ACHIEVING THE FIRST TWO GOALS.  
+
+See the LICENSE file distributed with this work for information
+regarding licensing, copying and copyright ownership.
+
```

Comment:
Reminder for us to update this when the other aspects have been settled. (closer to https://github.com/OISF/libhtp/blob/0.5.x/README but without the notice about Rust migration, I guess :P )